clients/src/test/java/org/apache/kafka/common/requests/JoinGroupRequestTest.java

```diff
@@ -64,7 +65,7 @@ public void shouldRecognizeInvalidCharactersInGroupInstanceIds() {
 
         for (char c : invalidChars) {
             String instanceId = "Is " + c + "illegal";
-            assertFalse(JoinGroupRequest.containsValidPattern(instanceId));
+            assertFalse(Topic.containsValidPattern(instanceId));
```

Comment:
This test case is duplicate to https://github.com/apache/kafka/blob/trunk/clients/src/test/java/org/apache/kafka/common/internals/TopicTest.java#L65. I feel it is fine to remove it.